posthog/src/main/java/com/posthog/PostHogInterface.kt

```diff
@@ -164,6 +164,16 @@ public interface PostHogInterface {
      */
     public fun distinctId(): String
 
+    /**
+     * Returns the [anonymousId] property
+     */
+    public fun anonymousId(): String
+
+    /**
+     * Returns the [isIdentified] property
+     */
+    public fun isIdentified(): Boolean
```

Comment:
@beradeep this should be private since its an implementation detail, no need to expose the `isIdentified` method

Reply:
but @marandaneto the `PostHogInterface` is a public interface, and Kotlin doesn't allow non-public interface members.

Reply:
Ah, okay 👍